resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -62,49 +137,300 @@
+        },
+        {
+            "Sid": "TransitGatewayRoutes",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:SearchTransitGatewayRoutes"
+            ],
+            "Resource": "arn:aws:ec2:*:*:transit-gateway-route-table/*"
+        },
+        {
+            "Sid": "Vpc",
+            "Effect": "Allow",
+            "Action": [
                 "ec2:DescribeVpcClassicLink",
                 "ec2:DescribeVpcClassicLinkDnsSupport",
                 "ec2:DescribeVpcEndpointConnectionNotifications",
                 "ec2:DescribeVpcEndpointServiceConfigurations",
-                "ec2:DescribeVpcEndpointServicePermissions",
                 "ec2:DescribeVpcEndpointServices",
-                "ec2:DescribeVpcPeeringConnections",
-                "ec2:DescribeVpnConnections",
-                "ec2:DescribeVpnGateways",
-                "ec2:GetAssociatedIpv6PoolCidrs",
-                "ec2:GetManagedPrefixListEntries",
-                "ec2:GetSerialConsoleAccessStatus",
-                "ec2:GetTransitGatewayAttachmentPropagations",
-                "ec2:GetTransitGatewayMulticastDomainAssociations",
-                "ec2:GetTransitGatewayPrefixListReferences",
-                "ec2:GetTransitGatewayRouteTableAssociations",
-                "ec2:GetTransitGatewayRouteTablePropagations",
-                "ec2:ModifyInstanceAttribute",
-                "ec2:SearchLocalGatewayRoutes",
-                "ec2:SearchTransitGatewayMulticastGroups",
-                "ec2:SearchTransitGatewayRoutes",
-                "ec2:StartNetworkInsightsAnalysis",
-                "elasticloadbalancing:ConfigureHealthCheck",
-                "kms:CreateGrant",
-                "s3:GetBucketTagging",
+                "ec2:DescribeVpcPeeringConnections"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VpcAttribute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVpcAttribute"
+            ],
+            "Resource": "arn:aws:ec2:*:*:vpc/*"
+        },
+        {
+            "Sid": "VpcEndpointService",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVpcEndpointServicePermissions"
+            ],
+            "Resource": "arn:aws:ec2:*:*:vpc-endpoint-service/*"
+        },
+        {
+            "Sid": "S3object",
+            "Effect": "Allow",
+            "Action": [
                 "s3:GetObjectAcl",
-                "s3:GetObjectTagging",
-                "s3:ListAllMyBuckets",
+                "s3:GetObjectTagging"
+            ],
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Sid": "Tiros",
+            "Effect": "Allow",
+            "Action": [
                 "tiros:CreateQuery",
                 "tiros:GetQueryAnswer",
-                "tiros:GetQueryExplanation",
+                "tiros:GetQueryExplanation"
+            ],
+            "Resource": "*"
```

Comment:
Is it possible to scope this to just Trios queries?

Reply:
looks like Tiros requires "* " for resources https://docs.aws.amazon.com/service-authorization/latest/reference/list_awstiros.html#awstiros-resources-for-iam-policies